intro_calculating_cred_grain.md

```diff
@@ -0,0 +1,162 @@
+# An introduction to calculating Cred and Grain
+
+I’m starting from the assumption that you have some familiarity with the SourceCred project. If you don’t I’d suggest you check out Dandelion's [20 minute talk (+Qs) from SustainWeb3](https://www.youtube.com/watch?v=yVTqRLekRl4), and some of the other documents in this repo.
+
+Overall, I’m going to throw around some math, but you should be able to get solid intuition on this if you’re comfortable talking about a [graph](https://en.wikipedia.org/wiki/Graph_theory) as a set of nodes and edges connecting them, where the edges can be directed and have weights. I’ll also talk some about [the PageRank algorithm](https://en.wikipedia.org/wiki/PageRank), for computing “importance” of nodes in a graph based on the “importance” of nodes connected to it, but I’ll try to motivate the intuition behind it as well.
+
+## The high level overview
+
+SourceCred takes information about contributions and generates a weights graph, and this post will first explain what that graph is and where it comes from. From this weights graph, it runs PageRank to generate a graph of Cred. This graph of Cred is then used to distribute grain. The grain can then be kept or traded, and soon will be able to be used to influence future graph weights. This process is diagrammed in the figure below.
+ 
+![](https://docs.google.com/drawings/u/1/d/s50mxZJ1DFD_yr4OFUxUd9g/image?w=551&h=250&rev=57&ac=1&parent=1kRxnuKAogf_cE0y6u2QYcjQimE0-jrBZ-LxcXKOUITw)
```

Comment:
We'll want to commit the diagram image in this repo, not a link out to Google docs.

Feedback on the diagram:

- The "contribute" and "data in location" sections are unclear -- I recommend using specific examples like GitHub and Discourse.
- Replace "weights graph" with "contribution graph"
- Replace "PageRank" with "CredRank" :D 
- Replace "cred graph" with "cred scores"

Reply:
fixed!  It (and the other images) now live in /docs/images, and I added the SVGs, so others can directly update them.